sdk/tables/azure-tables/src/index.ts

```diff
@@ -1,23 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-// This node built-in must be shimmed for the browser.
+import { TableServiceClient } from "./TableServiceClient";
 
-// This is a node dependency that needs to be replaced with a
-// different implementation in the browser.
-
-
-// this is a utility function from a library that should be external
-// for both node and web
-import { isNode } from "@azure/core-http";
-
-export function helloWorld(): string {
-  if (isNode) {
-    console.log("Node 👊");
-  } else {
-    console.log("Browser ❤");
-  }
-
-  return "Hello world!";
-
-}
+export * from "./generated/models";
```

Comment:
We may not want to export all the generated models. Can we do named exports instead?

```typescript
export { ModelA, ModelB, ModelC } from "./generated/models";
```